README.rst

```diff
@@ -10,76 +10,48 @@ Renku notebooks
     :target: https://conventionalcommits.org
 
 
-A simple external JupyterHub service, which provides interactive notebooks for
-the Renku platform.
+A simple service around the `Amalthea operator
+<https://github.com/SwissDataScienceCenter/amalthea>`_, which provides interactive Jupyter
+notebooks for the Renku platform.
 
-The service authenticates the user against JupyterHub and provides additional
-endpoints for launching notebooks for GitLab repository projects. An
-authenticated user may launch a notebook for any commit-sha in any project
-where they have developer access rights.
+The service relies on `renku-gateway <https://github.com/SwissDataScienceCenter/renku-gateway>`_
+for authentication. However, anonymous users are supported as well in which case anyone can
+start and use sessions for public renku projects. Therefore, the notebook service can run
+even without having the renku-gateway installed or present. In this case only sessions 
+for anonymous users can be launched.
 
 
 Endpoints
 ---------
 
-The service defines these endpoints:
+The service defines endpoints to list the active sessions for a user,
+start or stop a session. It can also provide information the logs from a running
+user sessions or information about work that was saved automatically if a user
+stops a sessions without commiting and pushing all their work to their project 
```

Comment:
```suggestion
start or stop a session. It can also provide the logs of a running
user session or information about work that was saved automatically if a user
stops a session without committing and pushing all their work to their project 
```